oio/xcute/common/job.py

```diff
@@ -17,20 +17,33 @@
 from oio.common.logger import get_logger
 
 
+class XcuteTask(object):
+
+    def __init__(self, conf, job_config, logger=None):
+        self.conf = conf
+        self.logger = logger or get_logger(self.conf)
+
+        self.tasks_per_second = int(job_config['tasks_per_second'])
+
+    def process(self, task_id, task_payload):
+        raise NotImplementedError()
+
+
 class XcuteJob(object):
 
     JOB_TYPE = None
+    TASK_CLASS = None
 
     DEFAULT_TASKS_PER_SECOND = 32
-    MAX_TASKS_BATCH_SIZE = 256
+    MAX_TASKS_BATCH_SIZE = 512
```

Comment:
That's still completely arbitrary.

Reply:
So what value do you propose? This value depends on the size of the task IDs and their payloads. These two sizes depending on the type of job. It is therefore impossible to have a default value that will be perfect for all jobs. If a job wants to change this value, it can override it in its class.

Reply:
If you want a default and not do it per class, imho:
- it should be commented to say why we have one, and the consequences
- isn't it too big? 512 allows <128 characters by task only